### ISSUE TITLE
Building in Visual Studio 2017 (15.7.4)

### ISSUE THREAD

**Issue by DJuego** [nabijaczleweli/pb-cpp#1]
Hi! Thank you for your contribution, @nabijaczleweli!  It seems... fun. :-) 

I am trying to build in Visual Studio 2017 (Windows 10 x64).

I get the following error:
```
1>------ Build started: Project: IMGUI_msvc2017, Configuration: Debug x64 ------
1>defaults.cpp
1>multibar.cpp
1>multibar_notifying_pipe.cpp
1>p:\mis-proyectos\profesional\imgui\inc\pb-cpp\src\multibar_notifying_pipe.cpp(61): error C2512: 'std::basic_ostream<char,std::char_traits<char>>': no appropriate default constructor available
1>c:\program files (x86)\microsoft visual studio\2017\enterprise\vc\tools\msvc\14.14.26428\include\iosfwd(601): note: see declaration of 'std::basic_ostream<char,std::char_traits<char>>'
1>progressbar.cpp
1>p:\mis-proyectos\profesional\imgui\inc\pb-cpp\src\progressbar.cpp(294): warning C4244: 'argument': conversion from 'const double' to 'size_t', possible loss of data
1>p:\mis-proyectos\profesional\imgui\inc\pb-cpp\src\progressbar.cpp(314): warning C4244: 'initializing': conversion from 'double' to 'size_t', possible loss of data
1>p:\mis-proyectos\profesional\imgui\inc\pb-cpp\src\progressbar.cpp(314): warning C4244: 'initializing': conversion from 'double' to 'const size_t', possible loss of data
1>generic.cpp
1>non_windows.cpp
1>windows.cpp
1>p:\mis-proyectos\profesional\imgui\inc\pb-cpp\src\util\windows.cpp(48): warning C4267: '-=': conversion from 'size_t' to 'SHORT', possible loss of data
1>simple.cpp
1>Generating Code...
1>Done building project "IMGUI_msvc2017.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

Any idea? I am not a C++ expert...

Thanks!

**Comment by nabijaczleweli** [comment]
If you would be so kind so as to try the current `master` branch (or just apply [this diff](https://github.com/nabijaczleweli/pb-cpp/commit/316c7a8819b1ffac5201e2b7bef8dd1961935e85#diff-c5481fb34a18cfbb8e5515c96bd23403L61) locally) and report back with whether it worked I will endeavour to release (or fix) the bugfix ASAP

**Comment by DJuego** [comment]
Yep! It works! @nabijaczleweli   I can confirm it. I have test any examples successfully! Thank you for the swift fix!

DJuego

**Comment by nabijaczleweli** [comment]
Thanks for the bug report and have fun using the library!

Released in [`v0.1.3`](https://github.com/nabijaczleweli/pb-cpp/releases/tag/v0.1.3).
